Sources/SnapshotTesting/Strategy/View.swift

```diff
@@ -104,42 +106,48 @@ private func addImagesForRenderedViews(_ view: View) -> [Async<View>] {
 
 fileprivate extension View {
   var snapshot: Async<Image>? {
-    #if os(iOS) || os(tvOS) || os(watchOS)
+    func inWindow<T>(_ perform: () -> T) -> T {
```

Comment:
This is required for views that snapshot themselves in order to "trick" it into rendering @2x. This logic used to be hard-coded to WKWebViews but I noticed that SceneKit/SpriteKit views were also rendering the wrong size.